api/gunicorn.conf.py

```diff
@@ -0,0 +1,46 @@
+wsgi_app = "conf.wsgi:application"
+capture_output = True
+accesslog = "-"
+errorlog = "-"
+chdir = "/api"
+bind = ["0.0.0.0:8000"]
+workers = 4
+timeout = 120
+logconfig_dict = {
+    # NOTE: Most of this is inherited from the default configuration
+    # https://github.com/benoitc/gunicorn/blob/cc2e3835784542e65886cd27f64d444309fbaad0/gunicorn/glogging.py#L48-L86
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "generic": {
+            "format": "[%(asctime)s - %(name)s - %(lineno)3d][%(levelname)s] %(message)s",  # noqa: E501
+        },
+    },
+    "root": {"level": "INFO", "handlers": ["console"]},
+    "loggers": {
+        "gunicorn.error": {
+            "level": "INFO",
```

Comment:
Shouldn't this be "ERROR"? It looks a bit strange when everything else suggests it should be so 🤔

Reply:
Very curious that it doesn't work otherwise.

Reply:
I think `gunicorn.error` is just the _name_ of the logger 🤷🏼‍♀️ not necessarily the level for it or related to levels in any way. It is definitely an odd name though!